slc/sample-app/thermostat/efr32/thermostat-thread.slcp

```diff
@@ -13,6 +13,10 @@ component:
   - id: device_init
   - id: sl_main
   - id: clock_manager
+  - id: common_token_manager_dynamic
+    condition: [device_series_3]
+  - id: common_token_manager
+    condition: [device_series_3]
```

Comment:
maybe this should be part of matter_platform_mg with the condition.

Reply:
The problem is that this is needed for the cmp at all times... So a good start is to have it here, but I will also add it in the zigbee_cmp file.